apps/projects/app/components/Content/IssueDetail/components/Detail.js

```diff
@@ -337,15 +373,16 @@ const Detail = ({
       <div style={{ flex: 1, maxWidth: '359px', width: '295px' }}>
         <DetailsCard>
           <FieldTitle>Activity</FieldTitle>
-          {Object.keys(issueEvents).sort((a,b) =>
-            new Date(a) - new Date(b)
-          ).map((eventDate, i) => {
-            return <IssueEvent key={i} {...issueEvents[eventDate]} />
+          {Object.keys(issueEvents).length > 0
+            ? Object.keys(issueEvents).sort((a,b) =>
+              new Date(a) - new Date(b)
+            ).map((eventDate, i) => {
+              return <IssueEvent key={i} {...issueEvents[eventDate]} />
+            })
+            : 'This issue has no activity'
```

Comment:
The text for the message initially was "This issue has no **bounty** activity" , I personally don't have inconvenience with the shorter text as long as @stellarmagnet is ok with the change